cli/cli_flags.go

```diff
@@ -0,0 +1,521 @@
+package cli
```

Comment:
Let's call this file `options.go` (to preserve the original name and avoid `cli/cli_xx`).